data/net/minecraft/world/level/gameevent/GameEvent.mapping

```diff
@@ -0,0 +1,115 @@
+CLASS net/minecraft/world/level/gameevent/GameEvent net/minecraft/world/level/gameevent/GameEvent
+	COMMENT Describes an in game event or action that can be detected by listeners such as the Sculk Sensor block.
+	FIELD BLOCK_ATTACH Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a block is attached to another. For example when the tripwire is attached to a tripwire hook.
+	FIELD BLOCK_CHANGE Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a block is changed. For example when a flower is removed from a flower pot.
+	FIELD BLOCK_CLOSE Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a block such as a door, drap door, or gate is closed.
+	FIELD BLOCK_DESTROY Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a block is destroyed or picked up by an enderman.
+	FIELD BLOCK_DETACH Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a block is detached from another block. For example when the tripwire is removed from the hook.
+	FIELD BLOCK_OPEN Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a block such as a door, trap door, or gate has been opened.
+	FIELD BLOCK_PLACE Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a block is placed in the world.
+	FIELD BLOCK_PRESS Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a block such as a button or pressure plate has been activated.
+	FIELD BLOCK_SWITCH Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a block such as a lever is switched on.
+	FIELD BLOCK_UNPRESS Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a block such as a button or pressure plate returns to it's unactivated state.
+	FIELD BLOCK_UNSWITCH Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a block such as a lever is turned off.
+	FIELD CONTAINER_CLOSE Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a block such as a chest, barrel, or shulker box is closed.
+	FIELD CONTAINER_OPEN Lnet/minecraft/world/level/gameevent/GameEvent;
+		COMMENT This event is broadcasted when a block such as a chest, barrel, or shulker box is opened.
```

Comment:
Maybe add a note about it being a block with an inventory?

Reply:
Unfortunately it's not that straight forward. There are many blocks with inventories such as droppers and hoppers which do not trigger this. The event is also broadcast when a chest minecart is accessed but possibly not when a horse with a chest is accessed. This is likely due to the system not being finished on Mojang's end. I will try to clarify this comment a bit.